src/components/Blog/ReleasePost.tsx

```diff
@@ -3,9 +3,13 @@ import { FunctionComponent } from 'react'
 import { MDXRemote } from 'next-mdx-remote'
 import Link from 'next/link'
 
+import { Alert } from '@components'
 import { PostComponentProps } from '@interfaces/posts'
 import { formatDate } from '@util'
 
+export type ReleaseComponents = import('mdx/types').MDXComponents
```

Comment:
Do you need to export this?

Reply:
As above, please see: 855da3d

Reply:
Awesome! Thanks @zlonko! Solid work on this!